backend/webapi/Models/Party.cs

```diff
@@ -33,6 +33,8 @@ public class Party : BaseAuditable
 
     public string? Cpn { get; set; }
 
+    public Instant? UserAccessAgreementDate { get; set; }
```

Comment:
I don't think we need to record this date on the `Party`, it is already on the User Access Agreement in the `AccessRequest` table